src/Sylius/Bundle/AddressingBundle/Form/Type/CountryChoiceType.php

```diff
@@ -40,10 +40,11 @@ public function __construct(RepositoryInterface $countryRepository)
      */
     public function configureOptions(OptionsResolver $resolver): void
     {
+        $defaultValue = null;
         $resolver
             ->setDefaults([
                 'choice_filter' => null,
-                'choices' => function (Options $options): iterable {
+                'choices' => function (Options $options) use (&$defaultValue): iterable {
```

Comment:
I'm not 100% sure if passing value by reference is the best approach, but I can't about anything else right now 👐

Reply:
Otherwise we would have to either reevaluate it and do it twice or do it before, but then we don't have the options there.